docs/source/intro.rst

```diff
@@ -25,9 +25,9 @@ of each implementation with the generic usage that Dask demanded. People may fin
 `code`_ which parses URLs and creates file-system
 instances interesting.
 
-.. _Dask: http://dask.pydata.org/en/latest/
-.. _docs: http://dask.pydata.org/en/latest/remote-data-services.html
-.. _code: https://github.com/dask/dask/blob/master/dask/bytes/core.py#L266
+.. _Dask: https://dask.pydata.org/en/latest/
+.. _docs: https://dask.pydata.org/en/latest/how-to/connect-to-remote-data.html
+.. _code: https://github.com/dask/dask/blob/7e4beffb/dask/bytes/core.py#L345
```

Comment:
Note that https://github.com/dask/dask/commit/7e4beffb is the last [`dask/bytes/core.py`](https://github.com/dask/dask/blob/main/dask/bytes/core.py) commit with function `expand_paths_if_needed` defined.

Next commit https://github.com/dask/dask/commit/77d51a1ae3abfff6645132c7100911a37369d58c removes `expand_paths_if_needed` from [`dask/bytes/core.py`](https://github.com/dask/dask/blob/main/dask/bytes/core.py).